### ISSUE TITLE
Change icons across the Dashboard with the new UI icons set (2.8)

### ISSUE THREAD

**Issue by clagom** [canonical-web-and-design/jaas-dashboard#453]
Now that we have new set of UI icons we can change the current ones on the Dashboard

CC @spencerbygraves @waynecrosby @barrymcgee @hatched

**Comment by cristinadresch** [comment]
@spencerbygraves do we have a list of new icons that should be applied to the dashboard?

**Comment by spencerbygraves** [comment]
@cristinadresch I'll add the new ones to the designs then supply the assets.

**Comment by cristinadresch** [comment]
sounds good @spencerbygraves let me know if you want to do that this/next week and ill assign this to you for this iteration

**Comment by spencerbygraves** [comment]
@cristinadresch yes, we can do it this iteration.

**Comment by spencerbygraves** [comment]
@barrymcgee here are the icons to be updated. As discussed earlier, perhaps this is for after 2.8.

- [Models](https://assets.ubuntu.com/v1/4e54368c-models.svg)
- [Controllers](https://assets.ubuntu.com/v1/aad9b854-controllers.svg)
- [User](https://assets.ubuntu.com/v1/5e03e442-user.svg)
- [Applications](https://assets.ubuntu.com/v1/4abf3881-applications.svg)
- [Units](https://assets.ubuntu.com/v1/921226e9-units.svg)
- [Machines](https://assets.ubuntu.com/v1/cc2e05a2-machines.svg)
- [Fullscreen](https://assets.ubuntu.com/v1/38ff5aeb-fullscreen.svg)
- [Close](https://assets.ubuntu.com/v1/13e6ccdd-close.svg)
- [Chevron](https://assets.ubuntu.com/v1/9056774c-chevron-up.svg)

**Model list**

![image](https://user-images.githubusercontent.com/9105999/82236021-6a0b7280-992b-11ea-908c-99399a4224d3.png)

**Model detail**

![image](https://user-images.githubusercontent.com/9105999/82236249-bc4c9380-992b-11ea-9be9-15434fe0e338.png)

**Modal**

![image](https://user-images.githubusercontent.com/9105999/82236316-d5554480-992b-11ea-8ffb-60e26940c3c6.png)

**Comment by cristinadresch** [comment]
Waiting on vanilla to update icons

**Comment by hatched** [comment]
Lets be sure we address this during this issue: https://github.com/canonical-web-and-design/jaas-dashboard/pull/506/files#diff-a9be8c3fe23b6d1f41c315d76fe0dafbR89

**Comment by spencerbygraves** [comment]
@bartaz do you know when all icons will be added to Vanilla?

**Comment by bartaz** [comment]
@spencerbygraves We don't have it really prioritized. If there is a need for that, we can schedule it.

**Comment by spencerbygraves** [comment]
@bartaz I think it would be good to schedule, now that we have the new set.

**Comment by bartaz** [comment]
Created separate issue in Vanilla to add JAAS (app) specific icons, not to wait for whole set getting into Vanilla: https://github.com/canonical-web-and-design/vanilla-framework/issues/3426

This should help us prioritize and get on it sooner.

**Comment by spencerbygraves** [comment]
@barrymcgee should I assign this to you now, or will you create a new issue when the Vanilla work is done?

**Comment by barrymcgee** [comment]
@spencerbygraves I've assigned this ticket to myself and will keep track of it.

**Comment by bartaz** [comment]
@barrymcgee We just released Vanilla 2.21.0 that contains all the icons (although they have to be included individually in SASS, [see the docs for details](https://vanillaframework.io/docs/patterns/icons#additional))
